### PR TITLE
Feature/input

### PR DESCRIPTION
**Pull request by mwj8410**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/mwj8410/Dais/pull/18" target="_blank">ReviewNinja

File: demo/dais.components.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/* global React */
+var InputContainer = React.createClass({
```
Comment (mwj8410) [inline demo/dais.components.js:2]: this is an example comment

Comment (mwj8410) [inline demo/dais.components.js:2]: !fixed

Comment (mwj8410) [inline demo/dais.components.js:2]: !fix

Comment (mwj8410) [inline demo/dais.components.js:2]: !fixed

**Pull request merged**